pages/_app.js

```diff
@@ -6,12 +6,57 @@ import useLocalStorageState from "use-local-storage-state";
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
 export default function App({ Component, pageProps }) {
+  const [settings, setSettings] = useLocalStorageState("settings", {
+    defaultValue: {
+      countryFlags: {
+        USD: true,
+        EUR: true,
+        GBP: true,
+        AUD: true,
+        CAD: true,
+        NZD: true,
+        CHF: true,
+        JPY: true,
+        CNY: true,
+      },
+      flagsTurnedOn: false,
+      alarmTriggerA: 5,
+      alarmTriggerB: 20,
```

Comment:
You are calling them `A` and `B` here but `1` and `2` elsewhere… maybe you can find a way to unify this.